projects/plugins/jetpack/extensions/shared/memberships/subscribers-affirmation.js

```diff
@@ -253,7 +253,7 @@ function SubscribersAffirmation( { accessLevel, prePublish = false } ) {
 
 	const reachForAccessLevel = getReachForAccessLevelKey( {
 		accessLevel,
-		emailSubscribers: emailSubscribersCount,
+		subscribers: emailSubscribersCount,
```

Comment:
For 

![image](https://github.com/user-attachments/assets/116643f7-deba-4f1d-8dd0-2e92bb307f61)